docs/api/concepts/craft.md

```diff
@@ -0,0 +1,45 @@
+# CRAFT
+
+CRAFT or Concept Recursive Activation FacTorization for Explainability is a method for automatically extracting human-interpretable concepts from deep networks.
+
+This concept activations factorization method aims to explain a pre-trained model's decisions both on a per-class and per-image basis by highlighting both "what".
+ 
+It is made up from 3 ingredients:
+1. a method to recursively decompose concepts into sub-concepts
+2. a method to better estimate the importance of extracted concepts
+3. a method to use any attribution method to create concept attribution maps, using implicit differentiation
+ 
+CRAFT requires splitting the model in two parts. So, we will prepare two functions $(g, h)$ such that $f(x) = (g \cdot h)(x)$. To put it simply, $g$ is the function that maps our input to the latent space (the penultimate layer of our model), and $h$ is the function that maps this penultimate layer to the output. It is important to note that if the model contains a global average pooling layer, it is strongly recommended to provide CRAFT with the layer before the global average pooling.
```

Comment:
I would change "CRAFT requires splitting the model in two parts. So, we will prepare two functions $(g, h)$ such that $f(x) = (g \cdot h)(x)$." to "CRAFT requires splitting the model in two parts: $(g, h)$ such that $f(x) = (g \cdot h)(x)$.". It seems clearer to me